.github/workflows/master_build_merge.yml

```diff
@@ -206,6 +211,48 @@ jobs:
             docker push ${DOCKERHUB_ORG}/$IMAGE:${LATEST_TAG}
             set +e
           }
+          elasticBuild() {
```

Comment:
This function is doing lot of things... broadly
1. setup of tools like go/mage, etc
& setting of configuration like PATH

2. Build docker image

3. tar file creation.


Split each into smaller functions and call them here. That way this function will be clear to understand.